### PR TITLE
add a section about GDPR relevant information

### PR DESCRIPTION
**Pull request by schiessle**
start with documentation about cookies, more can be added later. E.g. it might make sense to also have a few sentence about our GDPR tools

@NinaNC what do you think?
@rullzer can you double-check the technical details?

### CODE REVIEW

File: admin_manual/gdpr/cookies.rst [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+.. _`only data which contain personal data are relevant`: https://gdpr-info.eu/recitals/no-26/
+
+
+Cookies stored by Nextcloud
+===========================
+
+====================  ====================================  ================
+ Cookie                  Data Stored                             Lifetime
+====================  ====================================  ================
+ Session cookie        - session ID                          24 minutes
```
Comment (rullzer) [inline admin_manual/gdpr/cookies.rst]: So we store more cookies. But it is roughtly this list. Let me add some more info later. Also with examples what is stored and why.

**Comment by juliushaertl** [comment]
:ping_pong:

**Pull request merged**